src/rpc/rpc_reward_api.py

```diff
@@ -58,12 +60,12 @@ def get_rewards_for_cycle_map(self, cycle, expected_rewards=False):
                          .format(cycle, self.preserved_cycles, self.blocks_per_cycle,
                                  level_of_first_block_in_preserved_cycles, level_of_last_block_in_unfreeze_cycle))
 
-            # Decide on if paying actual rewards earned, or paying expected/ideal rewards
-            if expected_rewards:
```

Comment:
Is it correct that the naming in the code was inconsistent anyway? `expected_rewards` was used instead of the word `ideal_rewards` as the actual name derived from the configuration?

Reply:
On master branch currently, "expected rewards" and "ideal rewards" are equivalent and used interchangeably.

Reply:
Are all occurrences of expected rewards removed?

Reply:
Yes. I grepped for `expected` and I get a few matches from tzstats and tzkt api since some API calls are named "expected". Also the integration test for tzkt has "expected" string but it is referring to "expected results" compared to "actual results" so that's different from "expected rewards".